test/std-test.cc

```diff
@@ -10,6 +10,7 @@
 #include <string>
 #include <vector>
 
+#include "fmt/os.h"  // fmt::system_category()
```

Comment:
nit: we normally put symbol names without parentheses in comments like this

Reply:
Fixed.